concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -980,7 +980,7 @@ Hale-1973-1798-978	Food_and_Clothing-175	untie	v	07b3.41	352	UNTIE
 Hale-1973-1798-979	Tools_Instruments_Metals-1	tool	n	08a.01	1361	TOOL
 Hale-1973-1798-980	Tools_Instruments_Metals-2	stick	n	08a.02	1295	STICK
 Hale-1973-1798-981	Tools_Instruments_Metals-3	sling	n	08a.03	387	SLING
-Hale-1973-1798-982	Tools_Instruments_Metals-4	carrying pole	n	08a.04		
+Hale-1973-1798-982	Tools_Instruments_Metals-4	carrying pole	n	08a.04	3432	SHOULDER POLE
```

Comment:
I feel like "shoulder pole" is a hypernym of "carrying pole", so would unmap. But then again I don't know the exact intentions of the original data here.

Reply:
We have another _carrying pole_ gloss mapped to the concept so I'd leave it as is.